src/gsi/cplr_gfs_ensmod.f90

```diff
@@ -822,6 +822,10 @@ subroutine parallel_read_nemsio_state_(en_full,m_cvars2d,m_cvars3d,nlon,nlat,nsi
    if (k3u==0.or.k3v==0.or.k3t==0.or.k3q==0.or.k3oz==0) &  
       write(6,'(" WARNING, problem with one of k3-")')
 
+   do k=1,nsig
```

Comment:
@jderber-NOAA Is this change related?

Reply:
This change is not related, but fixed a bug in this routine.  It should have been in the master, since it was made in an earlier commit, but was not there and caused the regression tests to fail.  I will work with Mike to resolve it.

Reply:
@jderber-NOAA Thanks, good to know.  I'm fine with keeping it in this PR as long as it's here intentionally.